### ISSUE TITLE
Amazon Prime Video history page never loads, popup incorrect

### ISSUE THREAD

**Issue by profucius** [trakt-tools/universal-trakt-scrobbler#122]
I have built, tested and used the chrome extension for Netflix history, this is working properly. However when I click on the History icon for Amazon Prime Video, it loads two new tabs, one for the extension and one for prime video profile.

`chrome-extension://ckbdjkimpfiockafkkhhideggkafeopj/history.html#/amazon-prime`
`https://www.primevideo.com/settings?ref_=av_auth_return_redir`

The first tab just sits on a black screen with a loading icon spinning, indefinitely. I assume it is waiting on a response from the second tab. However, it seems to be unable to load any data from the second tab. Screenshot 1 below.

Below may be a separate issue that seems related but is only coincidental?

The second tab loads only the Account & Settings screen > Your account tab. It doesn't jump to the watch history tab automatically. Not sure if it needs to? Screenshot 2 below. However, when I go to the Watch history tab, it says there is no history, which I know to be false. Screenshot 3. Yet, if I navigate to the below URL, I see a full listing of my watch history as expected. Screenshot 4.

`https://www.amazon.com/gp/video/settings/watch-history`

Screenshot 1
![image](https://user-images.githubusercontent.com/18254607/153063149-bd2832c1-08b3-4c88-a508-c7758727ed34.png)

Screenshot 2
![image](https://user-images.githubusercontent.com/18254607/153063400-0bc3b632-9d23-49fc-9e97-da8d16366ccc.png)

Screenshot 3
![image](https://user-images.githubusercontent.com/18254607/153062749-7d056552-b7b5-4959-b651-7f04bc50e4cd.png)

Screenshot 4
![image](https://user-images.githubusercontent.com/18254607/153062911-da95198e-5719-457b-9147-69e3427941c1.png)

Is this a bug with the extension not loading the proper URL? Why does the first (extension) tab sit there loading indefinitely?

Note, I do all of my prime video watching via the android TV app on my TV, not within the browser. I want to be able to do post-syncing manually via this extension to keep the watch history in sync with Trakt.

**Comment by profucius** [comment]
Update: I dug into the src files (universal-trakt-scrobbler\src\services\amazon-prime\AmazonPrimeApi.ts) and this is on line 124 

`API_URL` = `'https://atv-ps.primevideo.com';`

However, going to that URL manually in a browser tab it produces a "HTTP ERROR 404" error. Perhaps this URL is no longer valid?

![image](https://user-images.githubusercontent.com/18254607/153446886-e9741724-b46c-4d79-91b8-021f804de476.png)

**Comment by MrMamen** [comment]
This is likely related to #117

**Comment by rafaelgomesxyz** [comment]
Closing as duplicate of #99.

**Comment by profucius** [comment]
This would seem like a duplicate ticket, however it is not; I am in a different region than the ticket you referenced. I tried the URL in that ticket and it does _not_ work for me. I am in the North America region.

What needs to be done differently for my region?

**Comment by rafaelgomesxyz** [comment]
I mean, it's still about Prime not working in a region, so the fix should be the same for both. I'll take a look into it later today and will let you know what I find.

**Comment by profucius** [comment]
I have commented within #117 about this issue, although I am not sure they are the same, as mine is specifically about USA region, not EU.
